.github/workflows/dev_on_workflow_tests_pro_e2e.yml

```diff
@@ -147,7 +147,7 @@ jobs:
           browser: chrome
           config-file: cypress/cypress.config.ts
           env: TAGS="@P0"
-          record: ${{ github.ref == 'refs/heads/master' }} # for Cypress Cloud
+          record: true #${{ github.ref == 'refs/heads/master' }} # for Cypress Cloud
```

Comment:
À enlever, non ?

Reply:
d'où le titre "To be reverted" mais comme ça ne marche pas, j'ai besoin de voir les exécutions dans Cypress Cloud